packages/react/src/components/Fieldset/Fieldset.mdx

```diff
@@ -1,22 +1,51 @@
-import { Meta, Controls, Primary } from '@storybook/blocks';
+import { Meta, Controls, Primary, Canvas } from '@storybook/blocks';
 import { CssVariables } from '@doc-components';
 import css from '@digdir/designsystemet-css/fieldset.css?inline';
 import * as FieldsetStories from './Fieldset.stories';
 
 <Meta of={FieldsetStories} />
 
 # Fieldset
+`Fieldset` brukes til å gruppere og navngi felt som naturlig hører sammen, for eksempel datofelt eller adressefelt. Komponenten hjelper med å organisere informasjon, gjøre skjemaer mer oversiktlige og forbedre tilgjengeligheten for skjermlesere.
 
 <Primary />
 <Controls />
 
-# Fieldset
+## Bruk
 
 `Fieldset` er et [fieldset](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset)-element med diverse hjelpfunksjoner for å hjelpe med bruk og gruppering av kontrollelementer i et skjema.
 
+## Eksempler 
+
+### Checkbox
+<Canvas of={FieldsetStories.WithCheckbox} />
+
+### Spørsmål som overskrift
+Å bruke selve spørsmålet som overskrift er god praksis i mange tilfeller, fordi brukere med skjermleser da bare hører spørsmålet én gang. 
+
+På sider med flere inndatafelt, gjør det det enklere for brukere med skjermleser om spørsmålet er en overskrift. Dette gjør det lettere å forstå at alle feltene som følger hører til samme gruppe som overskriften.
+<Canvas of={FieldsetStories.LegendAsHeading} />
```

Comment:
This felt kinda weird with and example using on legend, would like more context for this example.

Reply:
I agree, but we just did what govuk did: https://design-system.service.gov.uk/components/fieldset/